app/assets/stylesheets/theme/main-content.scss

```diff
@@ -3,7 +3,6 @@
   z-index: 0;
   padding-top: $spacer;
   padding-bottom: $spacer;
-  background: #eaedef;
-  background: $braveGradient-LegacyPink;
+  background: #d0d2dc;
```

Comment:
This might be the happiest day of my life. Can we try this color out `#eceff1`

Reply:
In Round 4